### PR TITLE
PLAT-1487 PLAT-1587 Updates to CLI

### PR DESCRIPTION
**Pull request by justin-shipstation**
- [PLAT-1487](https://auctane.atlassian.net/browse/PLAT-1487) - Remove getSalesOrdersByDate test from ShipEngine Connect
- [PLAT-1587](https://auctane.atlassian.net/browse/PLAT-1587) - `connect init` shouldn't create extra files for Order Apps

### CODE REVIEW

File: src/core/generators/apps-new.ts [hunk 1]
```diff
@@ -528,7 +514,7 @@ class AppsNew extends Generator {
     const dependencies: string[] = [];
     const devDependencies: string[] = [];
 
-    devDependencies.push("@shipengine/connect");
+    devDependencies.push("@shipengine/connect-sdk");
```
Comment (jeversmann-shipstation) [inline src/core/generators/apps-new.ts:517]: I think you'll need to update the template files to import from the correct library as well.

Comment (justin-shipstation) [reply src/core/generators/apps-new.ts:517]: Good Catch!

Comment (justin-shipstation) [reply src/core/generators/apps-new.ts:517]: @jeversmann-shipstation updated

**Review by rkrauskopf — CHANGES_REQUESTED** [review]
* Only use `@shipengine/connect` instead of `@shipengine/connect-sdk` in the generated order apps. See the [IBuy Sample app](https://github.com/ShipEngine/connect-samples/tree/main/iBuy) for reference.

* Why was the `getSalesOrdersByDate` integration test removed?

**Comment by justin-shipstation** [comment]
> * Only use `@shipengine/connect` instead of `@shipengine/connect-sdk` in the generated order apps. See the [IBuy Sample app](https://github.com/ShipEngine/connect-samples/tree/main/iBuy) for reference.

So the decision is to change its dependency to the sdk because that is what is actually necessary for the project to run, the connect umbrella comes with a lot of baggage that is not necessary for runtime. We have mitigated this up until now with an aliasing hack but we would like to clean this up moving forward.

> * Why was the `getSalesOrdersByDate` integration test removed?

This was removed at the request of @jeversmann-shipstation

**Comment by jeversmann-shipstation** [comment]
> Why was the getSalesOrdersByDate integration test removed?

Integration developers usually won't have the ability to make this test pass, and we don't want to prevent them from publishing or get them into the habit of skipping the tests every time.

**Comment by jeversmann-shipstation** [comment]
We should remove this test too, since not every order app needs to implement acknowledge:
```
acknowledgeOrders
     FAIL  it validates the acknowledgeOrders method with notifications notifications: [object Object]
            Error: Error in the acknowledgeOrders method. NotImplementedError
                at Object.error (/Users/jeversmann/.nvm/versions/node/v12.14.0/lib/node_modules/@shipengine/connect/node_modules/@shipengine/connect-sdk/src/internal/common/errors.ts:42:17)
                at OrderApp.acknowledgeOrders (/Users/jeversmann/.nvm/versions/node/v12.14.0/lib/node_modules/@shipengine/connect/node_modules/@shipengine/connect-sdk/src/internal/orders/order-app.ts:153:13)
```

**Comment by ricky** [comment]
development moved to https://github.com/ShipEngine/connect

**Pull request closed without merging**